test/dummies/vars.go

```diff
@@ -23,58 +23,61 @@ func GetYamlVal(variable string) string {
 }
 
 var ( // Declare exported dummy vars.
```

Comment:
Many of the updates to our dummies are about making the values actually compatible with k8s. Names and such used to have capitals or symbols in them.